src/ftxui/component/screen_interactive.cpp

```diff
@@ -858,6 +858,16 @@ void ScreenInteractive::Draw(Component component) {
   ResetCursorPosition();
   std::cout << ResetPosition(/*clear=*/resized);
 
+  // clear terminal output if screen dimx decreases
```

Comment:
It would be great explaining why (and not what) we do this.

What about:
```cpp
If the terminal width decrease, the terminal emulator will start wrapping lines and make the display dirty. We should clear it completely.
```

Reply:
Great explain, thanks